src/main/java/core/basesyntax/service/ReadFromFileService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface ReadFromFileService {
+    public String readFromFile(String pathToFile);
```

Comment:
Interface methods are public by default

Reply:
Agree with you, but I think that it's not making our code smells bad